_commontech/release_notes_for_customers.md

```diff
@@ -124,25 +124,25 @@ IBM Blue Box Cloud 2.1.0 comes pre-populated with Cirros 0.3.3 x86_64 images. Ad
 
 To use any of these images, additional information is provided in [this document](http://ibm-blue-box-help.github.io/help-documentation/gettingstarted/userguides/Cloud_Images_Provided_by_IBM/).
 
-#### **HIPAA and IBM Blue Box Dedicated Cloud**
+#### **HIPAA and IBM Bluemix Dedicated on OpenStack**
 
-	• You can select HIPAA Enablement for your IBM Blue Box Dedicated Cloud
+	• You can select HIPAA Enablement for your IBM Bluemix Dedicated on OpenStack
 
-A HIPAA-enabled cloud has the same configuration and expansion options as any other IBM Blue Box Cloud offering, but we are enabling _over 50 additional information-security controls_ that support our customers who must store personal health information (PHI). Because of this increased information security capability, a HIPAA-enabled cloud must be HIPAA-enabled from its inception: you cannot upgrade an existing cloud to enable HIPAA protocol. Also, you cannot downgrade from an IBM Blue Box Dedicated HIPAA cloud: the existing HIPAA cloud must be wiped—per HIPAA protocol—and you must order a new Dedicated cloud with non-HIPAA controllers.
+A HIPAA-enabled cloud has the same configuration and expansion options as any other IBM Bluemix on OpenStack offering, but we are enabling _over 50 additional information-security controls_ that support our customers who must store personal health information (PHI). Because of this increased information security capability, a HIPAA-enabled cloud must be HIPAA-enabled from its inception: you cannot upgrade an existing cloud to enable HIPAA protocol. Also, you cannot downgrade from an IBM Bluemix Dedicated on OpenStack HIPAA cloud: the existing HIPAA cloud must be wiped—per HIPAA protocol—and you must order a new Dedicated cloud with non-HIPAA controllers.
 
-Essentially, an IBM Blue Box Dedicated HIPAA cloud resembles any other Dedicated cloud, with these exceptions:
+Essentially, an IBM Bluemix Dedicated on OpenStack HIPAA cloud resembles any other Dedicated cloud, with these exceptions:
 
-	1.  Every IBM Blue Box Dedicated HIPAA cloud must have IBM Blue Box Dedicated Controllers.
+	1.  Every IBM Bluemix Dedicated on OpenStack HIPAA cloud must have IBM Bluemix Dedicated on OpenStack Controllers.
```

Comment:
Please refer to PR#421 for detailed re-wording of the lines in this area. Thanks!